app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -68,6 +40,7 @@ class MainActivity : AppCompatActivity() {
     private val orientationListener: OrientationEventListener by lazy { SmartOrientationListener(this) }
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        setupKoinFragmentFactory()
```

Comment:
Shouldn't this happen after the `super.` call?

Reply:
Docs show called before super. So that's intended.